### DISCUSSION TITLE
Create changelog website for Parcel

### DISCUSSION THREAD

**Discussion by JonasHiltl** [parcel-bundler/parcel#10001] [Show and tell]
Hi, I'm the creator of [Openchangelog](https://github.com/JonasHiltl/openchangelog) and a big fan of Parcel.

I created a changelog website for Parcel using Openchangelog which you can check out at [parcel.openchangelog.com](https://parcel.openchangelog.com/).
It automatically stays in sync with your `CHANGELOG.md` file.

Would love to have this linked in your CHANGELOG.md or docs to make it more accessible. After all who really wants to read a `CHANGELOG.md` file when they could have a beautiful, styled website.

**Comment by lorenzogrv** [comment]
Actually it's http status 500
